connectivity/drivers/emac/TARGET_Cypress/mbed_lib.json

```diff
@@ -1,3 +1,8 @@
 {
-    "name": "cypress-emac"
+    "name": "cypress-emac",
+    "target_overrides": {
+        "CYSBSYSKIT_01": {
+            "target.components_add": ["SCL"]
```

Comment:
Alternatively, combine [targets/TARGET_Cypress/TARGET_PSOC6/COMPONENT_SCL](https://github.com/ARMmbed/mbed-os/tree/master/targets/TARGET_Cypress/TARGET_PSOC6/COMPONENT_SCL) into [connectivity/drivers/emac/TARGET_Cypress/COMPONENT_SCL](https://github.com/ARMmbed/mbed-os/tree/master/connectivity/drivers/emac/TARGET_Cypress/COMPONENT_SCL) so the code isn't scattered in two places?